src/backend/PortabilityService.AnalysisService/appsettings.Development.json

```diff
@@ -0,0 +1,10 @@
+﻿{
+  "BlobStorageConnectionString": "UseDevelopmentStorage=true",
+  "CatalogStorageConnectionString": "UseDevelopmentStorage=true",
```

Comment:
If you put these in a `ConnectionStrings` object, you can use the more idiomatic `IConfiguration.GetConnectionString` in startup.

Reply:
Good points